app/docker/views/containers/create/createContainerController.js

```diff
@@ -280,11 +280,13 @@ angular.module('portainer.docker').controller('CreateContainerController', [
       }
       config.HostConfig.Dns = dnsServers;
 
+      var extraHosts = [];
```

Comment:
if we touch this line, can we do it something like this:
```
config.HostConfig.ExtraHosts = _.filter($scope.formValues.ExtraHosts, (v) => v.value)
```

Reply:
Thanks for the tip. Had to wrap in a map so we just grab the value from each object